test/jdk/ProblemList.txt

```diff
@@ -774,6 +774,9 @@ jdk/jfr/event/compiler/TestCodeSweeper.java                     8338127 generic-
 jdk/jfr/event/runtime/TestResidentSetSizeEvent.java             8309846 aix-ppc64
 jdk/jfr/jvm/TestWaste.java                                      8282427 generic-all
 
+java/lang/Thread/jni/AttachCurrentThread/AttachTest.java#id0    8343244 linux-all
+java/lang/Thread/jni/AttachCurrentThread/AttachTest.java#id1    8343244 linux-all
```

Comment:
yes, I think we have to temporarily exclude this test as it is failing frequently. The jdk_lang section of ProblemList is where java/lang/* tests are usually excluded.

Reply:
Thanks for the correction.
The test list has been move to jdk_lang section.